src/platform-apps/channels/message.tsx

```diff
@@ -53,13 +62,35 @@ export class Message extends React.Component<Properties> {
 
   renderMessage() {
     const parts = this.props.message.split(/(@\[.*?\]\([a-z]+:[A-Za-z0-9_-]+\))/gi);
-    return parts.map((part) => {
+    return parts.map((part, index) => {
       const match = part.match(/@\[(.*?)\]\(([a-z]+):([A-Za-z0-9_-]+)\)/i);
 
       if (!match) {
         return textToEmojis(part);
       }
 
+      if (match[2] === 'user') {
+        const profileId = this.getProfileId(match[3]);
+        const mention = `@${match[1]}`;
+        const props = {
+          className: 'message__user-mention',
+          key: match[3] + index,
+        };
+
+        if (profileId) {
+          return (
+            <ProfileLink
+              {...props}
+              id={profileId}
+            >
+              {mention}
+            </ProfileLink>
+          );
+        }
```

Comment:
sorry, the task description was not very good for this one! let's simplify this for now. *all* we need to do is have it highlighted. we don't need it to be "clickable" at all for now. it should just be styled with the accent background color, and then the font should be readable. in zero we have accent color and text on accent color in the theme. if we don't have that in our theme here, let's add it and make sure it works in light and dark mode.